account_asset_management/wizard/account_asset_compute.py

```diff
@@ -26,7 +26,7 @@ def asset_compute(self):
             module = __name__.split('addons.')[1].split('.')[0]
             result_view = self.env.ref(
                 '%s.%s_view_form_result'
-                % (module, self._name))
+                % (module, self._name.replace('.', '_')))
             self.note = _("Compute Assets errors") + ':\n' + error_log
```

Comment:
I suggest to fix the 
                % (module, self._name))
via
                % (module, self._table))

Reply:
Better indeed. Thanks.